src/main/java/redis/clients/jedis/JedisSentineled.java

```diff
@@ -8,13 +8,15 @@ public class JedisSentineled extends UnifiedJedis {
 
   public JedisSentineled(String masterName, final JedisClientConfig masterClientConfig,
       Set<HostAndPort> sentinels, final JedisClientConfig sentinelClientConfig) {
-    this(new SentineledConnectionProvider(masterName, masterClientConfig, sentinels, sentinelClientConfig));
+    super(new SentineledConnectionProvider(masterName, masterClientConfig, sentinels, sentinelClientConfig),
+        masterClientConfig.getRedisProtocol());
```

Comment:
good catch on extending protocol here. You might want this in a separate PR really, that way it appears as a bug fix so that people recognize in the release notes.

Reply:
#3692